build.assets/windows/build.ps1

```diff
@@ -473,7 +490,7 @@ function Write-Version-Objects {
     & $GoWinres simply --no-suffix --arch amd64 `
         --file-description "Teleport tsh command-line client" `
         --original-filename tsh.exe `
-        --copyright "Copyright (C) $Year Gravitational Inc." `
+        --copyright "Copyright (C) $Year Gravitational, Inc." `
```

Comment:
Best to run a tag build and then right click and go to properties on `tsh.exe` in a Windows environment. This version metadata is very picky and I seem to recall something about the comma breaking things.